### PR TITLE
modify create org regex, add default if not planFinishedAt

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
modify org name regex, add default value if planFinishedAt is nil

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/home-page-content/components/tableGroup/render.go [hunk 1]
```diff
@@ -270,6 +270,7 @@ func (t *TableGroup) addWorkbenchData(datas *apistructs.WorkbenchResponse, orgNa
 				},
 				OrgName: orgName,
 			}
+			issueItem.PlanFinishedAt = i18nLocale.Get(i18n.I18nKeyIssueNotSpecifiedDate)
```
Comment (chengjoey) [inline modules/openapi/component-protocol/scenarios/home-page-content/components/tableGroup/render.go:273]: image

File: pkg/erda-configs/i18n/workbench.json [hunk 2]
```diff
@@ -79,7 +79,8 @@
     "wb.content.component.issue.tomorrowExpired": "，明天到期:",
     "wb.content.component.issue.sevenDayExpired": "，7日内到期:",
     "wb.content.component.issue.thirtyDayExpired": "，30日内到期:",
-    "wb.content.component.issue.featureExpired": "，未来:"
+    "wb.content.component.issue.featureExpired": "，未来:",
+    "wb.content.component.issue.notSpecifiedDate": "未指定截止日期"
```
Comment (Effet) [inline pkg/erda-configs/i18n/workbench.json:83]: concat translated text is not a good practices, shall we support argument for i18n item, like:

`wb.content.component.issue.yourRemains: "当前你还有 %d，%d 个xxx，%d 个yyy"`

File: modules/core-services/utils/regexp.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ package utils
 
 import "regexp"
 
-var orgReg = regexp.MustCompile(`^[0-9a-z]\d*[a-z-][a-z\d-]*[0-9a-z]$`)
+var orgReg = regexp.MustCompile(`^(?:[a-z]+|[0-9]+[a-z]+|[0-9]+[-]+[a-z0-9])+(?:(?:(?:[-]*)[a-z0-9]+)+)?$`)
```
Comment (Effet) [inline modules/core-services/utils/regexp.go:18]: make a comment describe what this regexp means, it's too complex

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go [hunk 4]
```diff
@@ -102,7 +102,7 @@ func (o *OrgFormModal) SetProps() {
 				"rules": []interface{}{
 					map[string]interface{}{
 						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
-						"pattern": "/^(?![0-9]+$)[a-z0-9-]*$/",
+						"pattern": "/^(?![0-9]+$)^[^-](.*[^-])?$[a-z0-9-]*$/",
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: make a comment describe what this regexp means, it's too complex

**Review by Effet — CHANGES_REQUESTED** [review]

File: pkg/erda-configs/i18n/workbench.json (hunk 2 above)
Comment (chengjoey) [reply pkg/erda-configs/i18n/workbench.json:83]: these issue status used in different struct, so can`t make in one argument

File: modules/core-services/utils/regexp.go (hunk 3 above)
Comment (chengjoey) [reply modules/core-services/utils/regexp.go:18]: org regex modify from 
image
,but org name can not be pure digital

Comment (Effet) [reply modules/core-services/utils/regexp.go:18]: add into code, and use english

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go (hunk 4 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: same with orgReg, but Different front-end writing

**Review by Effet — APPROVED** [review]

**Pull request merged**